<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainDraftWindow</class>
 <widget class="QMainWindow" name="CMainDraftWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMainDraftWindow</string>
  </property>
  <widget class="QWidget" name="m_pCentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="buttonDraftPlayer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>500</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DRAFT PLAYER</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="m_pDraftBoardTab">
       <attribute name="title">
        <string>Draft Board</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableView" name="m_pDraftBoardTblView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_pAvailPlayerTab">
       <attribute name="title">
        <string>Available Players</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableView" name="m_pAvailPlayerTblView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_pAllPlayersTab">
       <attribute name="title">
        <string>All Players</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableView" name="m_pAllPlayersTblView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_pMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_pFileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m_pImportPlayers"/>
    <addaction name="m_pStartDraft"/>
   </widget>
   <widget class="QMenu" name="m_pEditMenu">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="m_pFilterAvailPlayers"/>
   </widget>
   <widget class="QMenu" name="m_pOptionsMenu">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <addaction name="m_pFileMenu"/>
   <addaction name="m_pEditMenu"/>
   <addaction name="m_pOptionsMenu"/>
  </widget>
  <action name="m_pImportPlayers">
   <property name="text">
    <string>Import Players from NFL.com</string>
   </property>
  </action>
  <action name="m_pStartDraft">
   <property name="text">
    <string>New Draft</string>
   </property>
  </action>
  <action name="m_pFilterAvailPlayers">
   <property name="text">
    <string>Filter Available Players</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
